web/libs/datamanager/src/components/Common/Tabs/Tabs.scss

```
@@ -0,0 +1,109 @@
+.tabs {
+  height: 36px;
+  display: flex;
+  align-items: stretch;
+  justify-content: space-between;
+  box-shadow: 0 -1px 0 #D9D9D9 inset;
+  background: var(--sand_200);
+
+  &__list {
+    display: flex;
+    min-width: 0;
+  }
+
+  &__item {
+    color: rgba(0, 0, 0, 0.5);
```

Flagging as opportunity for css var replacement. Further usage review needed.
```suggestion
    color: rgba(var(--black), 0.5);
```